evap/results/exporters.py

```diff
@@ -122,53 +133,56 @@ def filter_evaluations(semesters, evaluation_states, degrees, course_types, cont
                 continue
             if not evaluation.can_publish_rating_results and not include_not_enough_voters:
                 continue
-            results = OrderedDict()
+            results: OrderedDict[int, list[QuestionResult]] = OrderedDict()
             for contribution_result in get_results(evaluation).contribution_results:
                 for questionnaire_result in contribution_result.questionnaire_results:
                     # RatingQuestion.counts is a tuple of integers or None, if this tuple is all zero, we want to exclude it
+                    question_results: list[QuestionResult] = questionnaire_result.question_results
                     if all(
-                        not question_result.question.is_rating_question
-                        or question_result.counts is None
-                        or sum(question_result.counts) == 0
-                        for question_result in questionnaire_result.question_results
+                        not isinstance(question_result, RatingResult)
+                        or (question_result.counts is None or sum(question_result.counts) == 0)
```

Comment:
`or` should evaluate left to right here, so the old code without parenthesis should still be fine, right? In that case, I'd prefer that. With the parenthesis, the last `or sum(question_result.counts) == 0` is more hidden and might be overlooked by readers.